# String_Array.msg

string[] data

